scoring/views.py

```diff
@@ -407,28 +407,41 @@ def get_context_data(self, **kwargs):
         context["sections"] = sorted(
             sections.values(), key=lambda section: section["code"]
         )
+        context["comparisons"] = comparisons
+
+        for group in council.get_related_councils(5):
+            if group["type"].slug == "composite":
```

Comment:
why does this only apply if it's composite?

Reply:
We only want the 5 "overall" similar councils, not the other types of similarity.

Reply:
(I just copied this block, wholesale, from the caps app)